app/code/pagetype/Page.php

```diff
@@ -1,7 +1,41 @@
 <?php
 
 use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\TextareaField;
 
 class Page extends SiteTree
 {
+    /**
+     * Page database name
+     *
+     * @var string
+     */
+    private static $table_name = "PageField";
```

Comment:
Can remove this field as it should just default to `Page`

Reply:
Will do 👌🏻